File: server/PredlaganjeSaradnjeIRCDemo/PredlaganjeSaradnjeIRC.Data/Model/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace PredlaganjeSaradnjeIRC.Data.Model
{
    public class Employee
    {
        [Required]
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string? PersonalNumber { get; set; }
        public int? Experience { get; set; }
    }
}

File: client/fpis-app/src/model/Employee.ts
export class Employee{
    id:number;

    firstName:string;

    lastName:string;

    personalNumber:string;

    expirience:number;


    constructor(id:number,firstName:string,lastName:string,personalNumber:string,expirience:number){
        this.id=id;
        this.firstName=firstName;
        this.lastName=lastName;
        this.personalNumber=personalNumber;
        this.expirience=expirience;
    }

}